### PR TITLE
Adding Nydea

### PR DESCRIPTION
**Pull request by magikste**
Tell me how to fix those errors, because its so confuse

### CODE REVIEW

**Comment by lepon01** [comment]
Looks like the editor screwed itself up.
I'll take a look soon

**Comment by lepon01** [comment]
Where's your invite URL?

File: data/en/bots/330332878070218754.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+avatar: >-
+  https://cdn.discordapp.com/avatars/488805846499721216/9fdc16f903d7c2bdf9da3e9646095878.png?size=2048
+cover: null
+custom_path: !<tag:yaml.org,2002:js/undefined> ''
+description: ' Nydea  » Nydea have more than ten commands modules, it have all in one. The bot that you look in your dreams, it''s here.'
+github: null
+images: []
+link: '330332878070218754'
```
Comment (lepon01) [inline data/en/bots/330332878070218754.md]: This is not a link.

File: data/en/bots/330332878070218754.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+> DM work commands.
+
+> Commands needing mention an user, you can use instead `ID | USERNAME | MENTION`.
+
+> Multiple command aliases, watch aliases using `ny!alias <command>`
+
+> Basic commands like `help, ping, prefix` with a awesome function.
+
+> Unknow command **response**.
```
Comment (Andre601) [inline data/en/bots/330332878070218754.md]: Is that really a "feature"?
Imo should no bot really have such a thing...

Comment (lepon01) [reply data/en/bots/330332878070218754.md]: "Awesome ping"
"Awesome prefix"
"Awesome help"

Comment (Andre601) [reply data/en/bots/330332878070218754.md]: Awesome!

File: data/en/bots/330332878070218754.md (hunk 1 above)
Comment (lepon01) [reply data/en/bots/330332878070218754.md]: This ID isn't even the ID of your bot.

**Comment by lepon01** [comment]
[You can edit your page here](https://github.com/magikste/Discord_Fork/edit/v2/data/en/bots/330332878070218754.md)

**Pull request closed without merging**